README.md

```diff
@@ -122,6 +122,17 @@ Then delete the job:
 
     $ kubectl delete job benchmark
 
+## Envoy using BoringSSL QAT private key provider
+
+You can also build a version of Envoy using the BoringSSL QAT pricate key provider:
+
+    # docker image build -t envoy-boringssl-qat:devel -f Dockerfile.clr.boringssl.envoy .
+
+Quickly testing the created Docker container:
+
+    $ docker run --rm -ti -p 9000:9000 --privileged -v $(pwd)/cert.pem:/etc/envoy/tls/tls.crt -v $(pwd)/key.pem:/etc/envoy/tls/tls.key -v $(pwd)/examples/boringssl-envoy-conf.yaml:/etc/envoy/config/envoy-conf.yaml envoy-boringssl-qat:devel
```

Comment:
what if we create a launcher script for this. with that we can make the docker command to include all the device mounts , seccomp, and caps and avoid using privileged.

Reply:
Ok, added that now.

Reply:
I wrote a basic script for that purpose now.